scripts/target_disk_converter.rb

```diff
@@ -0,0 +1,46 @@
+require 'awesome_spawn'
+require 'fileutils'
+
+module Build
+  class TargetDiskConverter
+    attr_reader :disk
+
+    def initialize(disk)
+      @disk = disk
+    end
+
+    def fix_azure_disk
+      disk_type = AwesomeSpawn.run!("qemu-img info --output json #{disk}").output.match(/format\": "(.*)"/)[1]
+      convert_to_raw if disk_type == "vpc"
```

Comment:
Is it sometimes raw and sometimes vpc?

Reply:
The original script was being used for downstream as well, and upstream was raw and downstream was vpc. We no longer build azure downstream, but I left it, just in case.